apps/next-product-starter/src/pages/answer.tsx

```diff
@@ -1,147 +1,62 @@
 import * as React from 'react'
-import {get, isEmpty, keys} from 'lodash'
-import {useRouter} from 'next/router'
-import Layout from 'layouts'
-import Link from 'next/link'
-import EssayQuestion from 'components/forms/quiz/essay-question'
-import MultipleChoiceQuestion from 'components/forms/quiz/multiple-choice-question'
-import type {Question, Questions} from '@skillrecordings/types'
 
-type AnswerProps = {
-  questions: Questions
-}
-
-const Answer: React.FC<AnswerProps> = () => {
-  const router = useRouter()
-  const [currentQuestion, setCurrentQuestion] = React.useState<Question>()
-
-  React.useEffect(() => {
-    const param: any = get(router.query, 'question')
-    if (!isEmpty(param)) {
-      const question = get(questions, param)
-      setCurrentQuestion(question)
-    }
-  }, [router])
-
-  const QuestionToShow = () => {
-    if (!currentQuestion) {
-      return null
-    }
-    switch (currentQuestion.type as string) {
-      case 'multiple-choice':
-        return <MultipleChoiceQuestion question={currentQuestion as Question} />
-      default:
-        return <EssayQuestion question={currentQuestion as Question} />
-    }
+import {QuizAnswerPage} from '@skillrecordings/quiz'
+import {Questions} from '@skillrecordings/types'
+
+export async function getStaticProps() {
```

Comment:
this is a "big change" in that the page component expects the quiz data as a prop instead of as a global/const or whatever. In this case I'm just dumping static data in, but we could back this with a data source which would be rad af.